### PR TITLE
Add hidden div to top places component

### PR DESCRIPTION
**Pull request by Keqi**

### CODE REVIEW

File: src/components/top_places/top_places.hbs [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <section class="top_places{{#if centered}} top_places{{ centered }}{{/if}}">
   <button class="top_places__close icon-close-small" type="button">Close</button>
 
+  <div id="top_places-{{top_places_data_source}}" style="display:none"></div>
```
Comment (thomasthesecond) [inline src/components/top_places/top_places.hbs]: Would prefer this uses our utility class `is-hidden`. Does this get shown?

Comment (Keqi) [inline src/components/top_places/top_places.hbs]: Hey @thomasthesecond , 

This is something I talked about with @mkunkel. These divs are  in each one of these components so I would need to refactor them all. That would make sense but they are going to be removed once I am finished with Destinations-Next refactor.

Comment (Keqi) [inline src/components/top_places/top_places.hbs]: Is it okay for you to merge it that way?

Comment (thomasthesecond) [inline src/components/top_places/top_places.hbs]: Ok sounds good to me

**Comment by thomasthesecond** [comment]
👍

**Pull request merged**